frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -715,9 +735,25 @@ function* removeOldMeshesAndLoadUpdatedMeshes(
       yield* put(removeIsosurfaceAction(layerName, targetAgglomerateId));
     }
 
-    yield* call(loadCoarseMesh, layerName, newSourceAgglomerateId, sourceNodePosition);
+    // Segmentations with more than 3 dimensions are currently not compatible
```

Comment:
Could this be detected in the code, showing an error toast or logging to airbrake?

Reply:
see the other comment. a toast will be shown in that unlikely event.